jpsreport/CMakeLists.txt

```diff
@@ -39,9 +38,8 @@ set (header_files
     methods/Method_I.h
     methods/Method_J.h
     methods/ConfigData_DIJ.h
-    IO/OutputHandler.h
-    general/ArgumentParser.h
-    general/Macros.h
+        general/ArgumentParser.h
```

Comment:
Please indent according to the lines above.

Reply:
Done